src/Invoice.php

```diff
@@ -480,42 +498,31 @@ public function invoiceItems()
     public function subscriptions()
     {
         return Collection::make($this->invoiceLineItems())->filter(function (InvoiceLineItem $item) {
-            return $item->type === 'subscription';
+            return $item->isSubscription();
         })->all();
     }
 
     /**
-     * Get all of the invoice items.
+     * Get all of the invoice line items.
      *
+     * @param  array  $params
```

Comment:
```suggestion
     * @param  array{ending_before?: string, expand?: string[], limit?: int, starting_after?: string}  $params
```

https://github.com/stripe/stripe-php/blob/0c65d3f93e0332be9a2e7486c16941e6a177cf70/lib/Service/InvoiceService.php#L48-L65